docker/clickhouse_metrics.xml

```diff
@@ -0,0 +1,24 @@
+<yandex>
```

Comment:
Put in to subfolder please

Reply:
would it be fine under `/docker/conf/clickhouse_metrics.xml`?

Reply:
Yes

Reply:
I would kindly suggest `docker/compose/*`